samcli/lib/config/samconfig.py

```diff
@@ -73,7 +74,14 @@ def get_all(self, cmd_names, section, env=DEFAULT_ENV):
         env = env or DEFAULT_ENV
 
         self._read()
-        return self.document[env][self._to_key(cmd_names)][section]
+        if isinstance(self.document, dict):
+            params = self.document.get(env, {}).get(self._to_key(cmd_names), {}).get(section, {})
```

Comment:
You should put `self.document.get(env, {})` in a variable as you are accessing it 3 times.

Reply:
done